File: src/back/php/php/leibniz_formula.php
<?php

function leibniz_formula() {
    $s =0;
    for ($i=0; $i < pow(10,8); $i++) { 
        $s += pow(-1,$i)/(($i*2)+1);
    }
    return $s * 4;
}

$pi = leibniz_formula();
echo 'pi='.$pi."\n";

File: back/golang/leibniz_formula/leibniz_formula.go
package main

import (
	"fmt"
	"math"
)

func main() {
	fmt.Print("pi=", leibniz_formula(), "\n")
}

func leibniz_formula() float64 {
	var s = 0.0
	for i := 0; i < int(math.Pow(10, 8)); i++ {
		s += math.Pow(-1, float64(i)) / float64((i*2)+1)
	}
	return s * 4
}
